test/com/google/enterprise/adaptor/opentext/OpentextAdaptorTest.java

```diff
@@ -3049,6 +3067,520 @@ public void testDoEmail() throws IOException {
         responseMock.outputStream.toString("UTF-8"));
   }
 
+  @Test
+  public void testLastModifiedQuery() {
+    SoapFactoryMock soapFactory = new SoapFactoryMock();
+    OpentextAdaptor adaptor = new OpentextAdaptor(soapFactory);
+    AdaptorContext context = ProxyAdaptorContext.getInstance();
+    Config config = initConfig(adaptor, context);
+    config.overrideKey("opentext.src", "1234,5678");
+    adaptor.init(context);
+
+    String query = adaptor.getLastModifiedQuery();
+    assertTrue(query.indexOf("Location_ID1=1234") != -1);
```

Comment:
assertTrue(query, query.contains(...));

Here and down a little farther there are some more.